cmake/linux/toolchain-x86_64.cmake

```diff
@@ -14,9 +14,12 @@ set (TOOLCHAIN_PATH "${CMAKE_CURRENT_LIST_DIR}/../../contrib/sysroot/linux-x86_6
 
 set (CMAKE_SYSROOT "${TOOLCHAIN_PATH}/x86_64-linux-gnu/libc")
 
-set (CMAKE_C_FLAGS_INIT "${CMAKE_C_FLAGS} --gcc-toolchain=${TOOLCHAIN_PATH}")
```

Comment:
It doesn't make sense of have `CMAKE_C_FLAGS` prepended when initializing `CMAKE_C_FLAGS`. If `CMAKE_C_FLAGS` is defined beforehand, this statement is useless. So we either have  `set (CMAKE_C_FLAGS_INIT "--gcc-toolchain=${TOOLCHAIN_PATH}")` or `set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} --gcc-toolchain=${TOOLCHAIN_PATH}")` . I think the second one is better since we can provide more stuff via cmdline.